src/renderer/components/CloudVersionTargetForm.tsx

```diff
@@ -0,0 +1,317 @@
+          value={targets.selectedId}
+          showSearch
+          optionFilterProp="children"
+          allowClear={false}
+          loading={targets.loading}
+          virtual={process.env.NODE_ENV !== "test"}
+          disabled={
+            !versions.selectedId ||
+            !!targets.error ||
+            !!targets.loading ||
+            disabled
+          }
+          placeholder={
+            !versions.selectedId
+              ? t(`Select firmware to see available models`)
+              : versions.placeholder ?? t(`Select radio model`)
+          }
+          notFoundContent={null}
+        >
+          {targets.available?.map((tar) => (
+            <Select.Option key={tar.id} value={tar.id}>
+              {tar.name}
+            </Select.Option>
+          ))}
+        </Select>
+      </Form.Item>
+
+      <Divider />
+
+      <Form.List name="selectedFlags" initialValue={selectedFlags}>
+        {(fields, { add }) => (
+          <Form.Item label="Flags">
+            {fields.map((value, index) => (
+              <div key={value.key}>
+                <FormTag
+                  {...{
+                    value,
+                    index,
+                    flags,
+                    selectedFlags,
+                    updateSelectedFlags,
+                  }}
+                />
+              </div>
+            ))}
+
+            <Form.Item>
+              <Button
+                type="dashed"
+                onClick={() => add({ flag: undefined, value: undefined })}
+                block
+                icon={<PlusOutlined />}
+              >
+                Add Flag
+              </Button>
+            </Form.Item>
+          </Form.Item>
+        )}
+      </Form.List>
+    </Form>
+  );
+};
+
+const VersionFiltersDropdown: React.FC<{
+  filters: VersionFilters;
+  updateFilters: (newFilters: VersionFilters) => void;
+}> = ({ filters, updateFilters }) => {
+  const [visible, setVisible] = useState(false);
+  const { t } = useTranslation("flashing");
+
+  const filterNames: Record<keyof VersionFilters, string> = {
+    includePrereleases: t(`Include pre-releases`),
+  };
+
+  return (
+    <Dropdown
+      visible={visible}
+      trigger={["click"]}
+      onVisibleChange={(flag) => setVisible(flag)}
+      overlay={
+        <Menu>
+          {(Object.entries(filters) as [keyof VersionFilters, boolean][]).map(
+            ([key, value]) => (
+              <Menu.Item key={key}>
+                <Checkbox
+                  checked={value}
+                  onChange={(e) =>
+                    updateFilters({ ...filters, [key]: e.target.checked })
+                  }
+                >
+                  {filterNames[key]}
+                </Checkbox>
+              </Menu.Item>
+            )
+          )}
+        </Menu>
+      }
+    >
+      {/* eslint-disable-next-line jsx-a11y/anchor-is-valid, jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions */}
```

Comment:
Btw these lint errors are because you shouldn't be using an `<a>` link for this. Use a button or just underlined text etc.

Reply:
okay, use now ant button with link type